Firebase.ABTesting/source/Firebase.ABTesting/ApiDefinition.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Foundation;
+
+namespace Firebase.ABTesting {
+	// @interface FIRExperimentController : NSObject
+	[DisableDefaultCtor]
+	[BaseType (typeof (NSObject), Name = "FIRExperimentController")]
+	interface ExperimentController {
+		// +(FIRExperimentController * _Nonnull)sharedInstance;
+		[Static]
+		[Export ("sharedInstance")]
+		ExperimentController SharedInstance { get; }
+
+		// -(void)updateExperimentsWithServiceOrigin:(NSString * _Nonnull)origin events:(FIRLifecycleEvents * _Nonnull)events policy:(NSObject * _Nonnull)policy lastStartTime:(NSTimeInterval)lastStartTime payloads:(NSArray<NSData *> * _Nonnull)payloads;
+		[Export ("updateExperimentsWithServiceOrigin:events:policy:lastStartTime:payloads:")]
+		void UpdateExperiments (string origin, LifecycleEvents events, NSObject policy, double lastStartTime, NSData [] payloads);
+
+		// -(NSTimeInterval)latestExperimentStartTimestampBetweenTimestamp:(NSTimeInterval)timestamp andPayloads:(NSArray<NSData *> * _Nonnull)payloads;
+		[Export ("latestExperimentStartTimestampBetweenTimestamp:andPayloads:")]
+		double GetLatestExperimentStartTimestampBetweenTimestamp (double timestamp, NSData [] payloads);
+	}
+
+	// @interface FIRLifecycleEvents : NSObject
+	[BaseType (typeof (NSObject), Name = "FIRLifecycleEvents")]
+	interface LifecycleEvents {
+		// extern NSString *const _Nonnull FIRSetExperimentEventName;
+		[Field ("FIRSetExperimentEventName", "__Internal")]
```

Comment:
I would bind this on its own static class

```csharp
[Static]
interface DefaultLifecycleEventNames {

	[Field ("FIRSetExperimentEventName", "__Internal")]
	NSString SetExperiment { get; }

	// extern NSString *const _Nonnull FIRActivateExperimentEventName;
	[Field ("FIRActivateExperimentEventName", "__Internal")]
	NSString ActivateExperiment { get; }

	// extern NSString *const _Nonnull FIRClearExperimentEventName;
	[Field ("FIRClearExperimentEventName", "__Internal")]
	NSString ClearExperiment { get; }

	// extern NSString *const _Nonnull FIRTimeoutExperimentEventName;
	[Field ("FIRTimeoutExperimentEventName", "__Internal")]
	NSString TimeoutExperiment { get; }

	// extern NSString *const _Nonnull FIRExpireExperimentEventName;
	[Field ("FIRExpireExperimentEventName", "__Internal")]
	NSString ExpireExperiment { get; }
}
```

and I would add an `Advice` attribute to point this out

```csharp
[Advice ("You can use the default event name 'DefaultLifecycleEventNames.SetExperiment'.")]
[Export ("setExperimentEventName")]
string SetExperimentEventName { get; set; }
```